src/test/kotlin/racing/CarRacingTest.kt

```diff
@@ -1,44 +1,51 @@
 package racing
 
-import org.junit.jupiter.api.Assertions.assertEquals
-import org.junit.jupiter.api.Assertions.assertTrue
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.greaterThan
+import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Test
+import racing.domain.Car
 
 class CarRacingTest {
-    private val cars = listOf(Car("서진혁"), Car("카샤"), Car("nextstep"))
-    private val carRacing = CarRacing(cars, TRY_COUNT)
 
     @Test
     fun `차량 댓수와 repeat 횟수가 동일한지 확인한다`() {
-        carRacing.execute()
-
-        assertEquals(CAR_COUNT, cars.size)
+        assertThat(cars.size, `is`(CAR_COUNT))
         cars.forEach {
-            assertEquals(TRY_COUNT, it.records.size)
+            assertThat(it.getRecords().size, `is`(TRY_COUNT))
```

Comment:
`hamcrest`이 없어도 아래와 같이 작성이 가능한데요!
혹시 추가하신 다른 이유가 있을까요?

```suggestion
            assertThat(it.getRecords().size).isEqualTo(TRY_COUNT)
```

Reply:
@vsh123 평소 hamcrest를 잘 사용하지 않았었는데요.
https://github.com/next-step/kotlin-racingcar/pull/220#discussion_r589339380 
말씀 주신 링크를 보고, 조금 사용해보려고 했었습니다.

Reply:
아하!! 제가 준 링크가 hamcrest를 이용한 예시여서 그랬군요!!
[이 링크](https://pjh3749.tistory.com/241)가 조금 더 제 의도와 맞을 것 같아요!! hamcrest를 쓴다기 보다는 assertThat을 사용했을 때의 가독성을 강조드리고 싶었습니다..! 혼란을 드려 죄송해요ㅠㅠ

Reply:
assertThat이 왼쪽에서 오른쪽으로 자연스럽게 읽혀 가독성이 높다는 말씀이시죠?
이해하였습니다! 감사합니다 :)